taskcluster/test/conftest.py

```diff
@@ -0,0 +1,74 @@
+from copy import deepcopy
+import pytest
+import requests_mock
+
+from taskgraph.generator import TaskGraphGenerator
+from taskgraph.parameters import parameters_loader
+from translations_taskgraph.util.substitution import substitute
+
+
+# These fixtures are largely cribbed from Gecko:
+# https://searchfox.org/mozilla-central/source/taskcluster/test
+@pytest.fixture(scope="session")
+def create_tgg():
```

Comment:
Thought: I guess doing fixtures this way means that type hinting won't work. I'm guessing this is a pythonic practice, but I'm wondering if it would evolve with adding more robust typing requirements.

Reply:
Yeah, this is quite standard pytest stuff (which probably predates `typing`...). I don't know of a way for fixtures to find their types based on where they are created, but I can update this to at least specify an explicit type for `create_tgg` where it's used.